xml/admin_ds_backup.xml

```diff
@@ -140,5 +142,92 @@
     </para>
    </listitem>
   </itemizedlist>
+ </sect2>
+ </sect1>
+ <sect1 xml:id="restore-ceph">
+  <title>Restoring a Ceph Node</title>
+  <para>
+   The basic procedure for recovering a node from backup is to reinstall the
+   node, replace its configuration files, and then re-orchestrate the cluster
+   so that the replacement node is re-added. 
+  </para>
+  <orderedlist>
+   <listitem>
+    <para>
+     Re-install the node. (See <link
+     xlink:href="https://documentation.suse.com/ses/6/single-html/ses-deployment/#ceph-install-stack"/>
+     steps 1-4.)
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Update to the latest patch level (See: <link
+      xlink:href="https://documentation.suse.com/sles/15-SP1/single-html/SLES-admin/#sec-zypper-softup-update"/>.)
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Install &salt; and &deepsea;. (See <link
+      xlink:href="https://documentation.suse.com/ses/6/single-html/ses-deployment/#ceph-install-stack"/>
+     steps 5-7.)
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Restore the old, saved &deepsea; and &salt; configuration files.
+    </para>
+    <para>
+     It may be necessary to set the permissions appropriately on the replaced
+     files and manually enable services. For instance:
+<screen>&prompt.smaster; chown salt:root /etc/salt/pki/master/master.pem
+&prompt.smaster; chown salt:root /etc/salt/pki/master/master.pub
+&prompt.smaster; chown -R salt:salt /srv/pillar/ceph/
+&prompt.smaster; systemctl enable salt-api
+&prompt.smaster; systemctl start salt-api
+&prompt.smaster; systemctl enable salt-minion
+&prompt.smaster; systemctl start salt-minion
+&prompt.smaster; systemctl enable salt-master
+&prompt.smaster; systemctl start salt-master</screen>     
```

Comment:
It's failing because these screen tags (182 - 190) are within a <para> element. This is normal for docbook, but geekodoc hates it. Just move it out :)